### PR TITLE
jsdoc: fix existing jsdoc and make jscs checks existence of jsdoc on functions

### PR DESCRIPTION
**Pull request by shaoshing**
**require `npm install`**

This PR fixed the existing jsdoc, to allow us enable the "enforceExistence" option

* corrected wrong jsdoc format
* added @ignore tag to those functions without jsdoc

"enforceExistence" option will not check:

* function(s) within documented function.
* fluxxor and react lifecycle functions (listed in the "enforceExistenceExcept" options).
* functions defined in test files.

### CODE REVIEW

File: src/js/jsx/tools/GuidesOverlay.jsx [hunk 1]
```diff
@@ -136,7 +136,11 @@ define(function (require, exports, module) {
             this.drawGuideEdges();
         },
 
-        // Draws the guide edge areas
+        /**
+         * Draws the guide edge areas
+         * 
+         * no @return
```
Comment (iwehrman) [inline src/js/jsx/tools/GuidesOverlay.jsx]: Just delete this line?

File: src/js/actions/shortcuts.js [hunk 2]
```diff
@@ -139,7 +139,7 @@ define(function (require, exports) {
     var beforeStartup = function () {
         var shortcutStore = this.flux.store("shortcut"),
             controller = this.controller;
-
+
```
Comment (iwehrman) [inline src/js/actions/shortcuts.js]: :space_invader:

**Comment by iwehrman** [comment]
Two micro nits, then I'm ready to do it! :heart_eyes:

**Comment by shaoshing** [comment]
Thanks @iwehrman . nits destroyed!

**Comment by shaoshing** [comment]
giphy

**Comment by iwehrman** [comment]
:hamburger: :fries:

**Pull request merged**